File: 프로그래머스/1/12917. 문자열 내림차순으로 배치하기/문자열 내림차순으로 배치하기.py
def solution(s):
    up,lo=[],[]
    for c in s:
        if c.isupper():
            up+=[c]
        else:
            lo+=[c]
    up=sorted(up,reverse=True)
    lo=sorted(lo,reverse=True)
    r=''.join(lo+up)
    return r

File: 프로그래머스/1/12917. 문자열 내림차순으로 배치하기/문자열 내림차순으로 배치하기.java
import java.util.Arrays;
import java.util.Collections;
class Solution {
    public String solution(String s) {
        String rlt = "",tmp[] = s.split("");
        Arrays.sort(tmp,Collections.reverseOrder());
        rlt=String.join("",tmp);
        return rlt;
    }
}